common-js/build.gradle

```diff
@@ -1,28 +1,84 @@
+ext {
+    kotlinx_html_version = "0.6.4" 
+}
+
+
 buildscript {
+    ext.kotlin_version = '1.2.60'
```

Comment:
This should be available globally, so you don't need this

Reply:
If that's not the case, then in the root gradle file we should add an `allprojects { ... }` section with `buildscript { ext { kotlin_version = '1.2.60' } }` in it. There's no reason why our kotlin version shouldn't stay consistent across all "projects"